---
title: "Project 2. Proline catabolism as a vulnerability in hemizygous PTEN deleted cancers."
collection: talks
type: "Talk"

---
In addition to OxPhos complexes I and II, dihydroorotate dehydrogenase (DHODH) and proline dehydrogenase (PRODH) reduce ubiquinone upstream of complex III and IV in the inner mitochondrial membrane. Within the proline metabolic pathway (Figure 3b), loss-of-function ALDH4A1 mutations result in the accumulation of its nucleophilic substrate pyrroline-5-carboxylate (P5C), which depletes the essential cofactor pyridoxal-5’-phosphate (PLP) through Knoevenagel condensation, causing selective cytotoxicity to cells of the central nervous system and connective tissue (Figure 3d). Conversely, biallelic mutations in ALDH18A1 result in proline auxotrophy and toxicity to the same tissue. Using novel selective lethal inhibitors of OxPhos complex IV, we previously found that ALDH18A1 and ornithine aminotransferase (OAT), which produce the reactive intermediate P5C from glutamate or ornithine are downregulated on both the protein and transcript level across cell lines that were resistant to complex IV inhibitor induced cell death, whereas the enzymes catalyzing the reverse catabolic reactions (ALDH4A1 and PRODH) were upregulated (Figure 3c). Consistently, resistant cell lines were hemizygous for chr10q, encoding both ALDH18A1 and OAT adjacent to the tumor suppressor PTEN (Figure 3a), which occurs in ~15-55% of prostate, skin and central nervous system cancers. Importantly, this relationship is maintained in patient samples taken from the cancer genome atlas, where PTEN copy number is significantly correlated with the expression of ALDH18A1 (Figure 3a). Moreover, chr10q loss decreases expression of an ALDH18A1 spliceoform (Figure 3e) required for ornithine synthesis from glutamate and ammonia detoxification, which is insensitive to allosteric inhibition by ornithine (Figure 3b). Taken together this data suggests that PTEN deletions alter the dosage of essential proline metabolic enzymes and restricts proline metabolism, providing a rationale therapeutic strategy to target PTEN hemizygous cancers through drug induced haploinsufficiency in ALDH18A1, and/or P5C mediated metabolic toxicity. 

![Alt text](/images/Figure_3_h.png)
------
Figure 3: Proline metabolism in PTEN deleted cancers. a, Chromosomal location of ALDH18A and OAT relative to PTEN (Top) and scatterplot of PTEN copy number versus ALDH18A1 mRNA expression across patient samples from TCGA. b, Proline metabolic pathway. Alternative splicing of ALDH18A1 alters allosteric feedback by ornithine. c, Differentially expressed proline metabolic proteins and transcripts across cell lines sensitive (red) and resistant (blue) to OxPhos complex IV inhibitors (shown in Figure 2c) taken from unenriched proteomics and RNA-seq data across five sensitive and five resistant cancer cell lines (P<1e-10). Values represent z-scored mean (n=3 biological replicates) of each cell line. d, Structure of P5C and PLP depletion in ALDH4A1 related metabolic disease. e, Alternative splice donor in Exon 2 of ALDH18A1 detected by mixture of isoform analysis (MISO). The percentage reads spliced in/posterior distribution shown right. f, Crystal structure of ornithine insensitive ALDH18A1 isoform (tan, pdb 7WX3), bound to ADP (green) and ɣ-glutamyl phosphate (blue). Two amino acid ALDH18A1 insertion that removes ornithine sensitivity shown in red. g, Example cofactor-substrate mimetic targeting the ALDH18A1 isoform lacking two base pair insertion. h, Diagram depicting proposed small-molecule high content competition screen. Pooled isogenic chr10q diploid (red nucleus) and hemizygous cells (green nucleus), will be treated with compounds, and the relative ratio of these cells will be kinetically monitored by fluorescence microscopy in the presence or absence of proline
